tox.ini

```diff
@@ -20,7 +20,7 @@ deps =
     -U
     git+https://github.com/Zac-HD/hypothesmith.git
 commands =
-    python -m unittest discover tests --catch {posargs}
+    python -m unittest discover tests --catch {posargs: -v}
```

Comment:
@Zac-HD Please let me know if you'd like me to accomplish this in another way, or if you want to drop it from this PR. I think these tests should be run in some sort of verbose mode on CI, because a lot of them are conditionally skipped (at least at the moment), and it's very hard to tell which ones those are in the current running mode.

Reply:
That looks great to me!  The status quo is mostly because I am a total novice with `unittest`, when I'm not writing for upstream I'd just use `pytest` :sweat_smile: